sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -2034,7 +2041,7 @@ class DataFrameSuite extends QueryTest with SharedSparkSession {
       .select(when($"cond", array(lit("a"), lit("b"))).otherwise($"a") as "res")
       .select($"res".getItem(0))
     def mapWhenDF: DataFrame = sourceDF
-      .select(when($"cond", map(lit(0), lit("a"))).otherwise($"m") as "res")
+      .select(when($"cond", functions.map(lit(0), lit("a"))).otherwise($"m") as "res")
```

Comment:
to avoid naming conflicts, how about rename `AdaptiveSparkPlanHelper.map` to `mapPlans`?